openfecwebapp/templates/partials/candidate/itemized-disbursements.html

```diff
@@ -0,0 +1,68 @@
+{% import 'macros/missing.html' as missing %}
+{% import 'macros/null.html' as null %}
+{% import 'macros/cycle-select.html' as select %}
+
+<section id="section-3" role="tabpanel" aria-hidden="true" aria-labelledby="section-1-heading">
+  <div class="heading--section">
+    <h2>Itemized Disbursements</h2>
+  </div>
+
+  <div class="content__section">
+    {{ select.cycle_select(election_years, cycle, duration=duration, id="cycle-1") }}
```

Comment:
The cycle select shouldn't have the option to do "full cycle" since we only have up to two years data at a time:
![image](https://cloud.githubusercontent.com/assets/1696495/23723762/44445cfc-03ff-11e7-90da-53634c475cad.png)